website/docs/r/kubernetes_cluster.html.markdown

```diff
@@ -54,6 +54,26 @@ resource "azurerm_kubernetes_cluster" "test" {
     Environment = "Production"
   }
 }
+
+output "id" {
+    value = "${azurerm_kubernetes_cluster.test.id}"
+}
+
+output "client_key" {
+  value = "${azurerm_kubernetes_cluster.test.kube_config.0.client_key}"
+}
+
+output "client_certificate" {
+  value = "${azurerm_kubernetes_cluster.test.kube_config.0.client_certificate}"
+}
+
+output "cluster_ca_certificate" {
+  value = "${azurerm_kubernetes_cluster.test.kube_config.0.cluster_ca_certificate}"
+}
+
+output "host" {
+  value = "${azurerm_kubernetes_cluster.test.kube_config.0.host}"
+}
```

Comment:
I'm going to push a commit to update the test to ensure these values are actually set